client/src/components/AdvisorReports/AdvisorReportsTable.js

```diff
@@ -47,11 +46,11 @@ const AdvisorReportsTable = ({ pageDispatchers, columnGroups, orgUuid }) => {
   }
 
   const advisors = data.advisorReportInsights
-  const rows = advisors.map(advisor => (
+  const rows = advisors.map((advisor, index) => (
     <AdvisorReportsRow
       row={advisor}
       columnGroups={columnGroups}
-      key={_uniqueId(`${advisor.uuid}_`)}
+      key={`${advisor.uuid}_${index}`}
```

Comment:
`advisor.uuid` is already a unique value

Reply:
The same advisor may appear more than once if they've moved organisations I believe.

Reply:
```
  const { done, result } = useBoilerplate({
    loading,
    error,
    modelName: "Organization",
    uuid: orgUuid,
    pageDispatchers
  })
  if (done) {
    return result
  }

  const advisors = data.advisorReportInsights
  const rows = advisors.map((advisor, index) => (
    <AdvisorReportsRow
      row={advisor}
      columnGroups={columnGroups}
      key={`${advisor.uuid}_${index}`}
    />
  ))
```
I thought we can't have duplicate advisors from single organization, or no ?